contracts/governed-owner/proxies/OwnerProxyBase.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity >=0.8.0;
+
+import "../SimpleGovernance.sol";
+
+abstract contract OwnerProxyBase {
+    SimpleGovernance public gov;
+
+    function initGov(SimpleGovernance _gov) public {
```

Comment:
Anyone can init?

Reply:
no security risk, but will add restriction to deployer only